.github/workflows/system-tests-opensource.yml

```
@@ -190,6 +190,8 @@ jobs:
     - name: Prepare system tests env
       run: |
         echo "MLRUN_DBPATH: http://$(minikube ip):${MLRUN_API_NODE_PORT}" > tests/system/env.yml
+        echo "MLRUN_SYSTEM_TESTS_GIT_TOKEN: ${{ secrets.SYSTEM_TEST_GITHUB_ACCESS_TOKEN }}" > tests/system/env.yml
```

Use append so that we don't override `MLRUN_DBPATH`
```suggestion
        echo "MLRUN_SYSTEM_TESTS_GIT_TOKEN: ${{ secrets.SYSTEM_TEST_GITHUB_ACCESS_TOKEN }}" >> tests/system/env.yml
```